### ISSUE TITLE
CI Migration: Update PRs with link to frontend IP

### ISSUE THREAD

**Issue by murog** [GoogleCloudPlatform/bank-of-anthos#274]
## Description
Blocked by #270 
After deploying a pr to a test namespace, it would be nice to update the pr with a link to the exposed ip for manual testing.  This is not currently a first party feature and would require working with the Github API

## Expected Behavior
* PR is opened
* CI runs
* CI deploys to test cluster
* Cloud Build comments / updates PR description with link to exposed IP

**Comment by askmeegs** [comment]
duplicate of #320
